### DISCUSSION TITLE
fragments offset

### DISCUSSION THREAD

**Discussion by demuellae** [GreenleafLab/ArchR#1262] [Questions / Documentation]
Hi Jeff and Ryan,
Hope you are doing well. Quick question: 
If you extract fragment GRanges using the "getFragmentsFrom(Arrow|Project)" function: Are the ends already offset to account for Tn5 insertion site centering (+/-4/5 bp)?

Best,
Fabian

**Comment by rcorces** [comment]
fragments are defined as being offset adjusted
https://www.archrproject.com/bookdown/a-brief-primer-on-atac-seq-terminology.html
